### PR TITLE
Update logging messages(#282)

### PR DESCRIPTION
**Pull request by FabioRosado**
# Description

Added some logging separation throughout the code, replaced concatenation with string formatting using %s as discussed in gitter. 
Please let me know if things are okay or if I should change something around.

**Fixes:** #282

## Status
**READY** | ~~**UNDER DEVELOPMENT**~~ | ~~**ON HOLD**~~

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation (fix or adds documentation)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested locally using unit tests
- [x] Tested using Travis on own fork

# Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
Nice one! I'm guessing the decrease in coverage is related to adding extra logging in an uncovered area, so that's fine. 

I'll take a look through later just to be sure I agree with the levels.

**Comment by FabioRosado** [comment]
I added a logging.error message in the welcome message but it seems that the ones not being logged are in loader (even though previous builds show that this one wasn't tested?) Let me know if this is the issue.

File: opsdroid/__main__.py [hunk 1]
```diff
@@ -104,7 +104,8 @@ def welcome_message(config):
                          "releases")
             _LOGGER.info("=" * 40)
     except KeyError:
-        pass
+        _LOGGER.error("'welcome-message: true/false' is missing in "
```
Comment (jacobtomlinson) [inline opsdroid/__main__.py]: I would make this a warning as it still works without it.

**Comment by FabioRosado** [comment]
I've made a test to try and bump the coverage up again but it kept failing. 

What I tried: 
* Use the test_loader.test_load_modules
* Mock the logging.error
* set module = None
* call  loader._load_modules(modules_type, modules)

The else statement doesn't run because of the for loop on line 223 of opsdroid.loader  causes the test to get a `TypeError: 'NoneType' object is not iterable` 

Will this cause an issue in the future and should we change the _load_modules function if into a try?

**Comment by jacobtomlinson** [comment]
Yeah don't worry for now. I'm happy to merge it even with the slight drop. Perhaps investigate this further in a separate issue.

**Pull request merged**